.github/workflows/ci.patch.yml

```
@@ -51,6 +51,14 @@ jobs:
     steps:
       - run: 'echo "No build required"'
 
+  install-from-lockfile-no-cache:
+    name: Install --locked --path ./zebrad/ zebrad, stable on ubuntu-latest, no cache
```

Can we remove some less important info in the job name?

```suggestion
    name: Install --locked zebrad on ubuntu-latest, no cache
```